### ISSUE TITLE
When using remote container, we're forced to download docker desktop, it is possible to use remote docker container via ssh only?

### ISSUE THREAD

**Issue by Bovey0809** [microsoft/vscode-remote-release#5307]
When using remote container, we're forced to download docker desktop, it is possible to use remote docker container via ssh only?

**Comment by chrmarti** [comment]
This is currently not possible. It might become available with https://github.com/microsoft/vscode-remote-release/issues/2994, but I'll keep this issue here to track this specific request.

**Comment by KonanTheLibrarian** [comment]
YES  [WSL runs 10x better without Docker Desktop link](Here https://nickjanetakis.com/blog/install-docker-in-wsl-2-without-docker-desktop)   
WSL can run on vanilla Docker and runs faster without crashing unfortunately VSCODE keeps trying to install newer "Docker" but then when you say "YES"  changes its mind and tries to force "Docker Desktop" instead which is NOT docker but very bad bug riddled wrapper for docker that fails integration and is crashy.  SERIOUS: Docker Desktop crashes so much that it is setup so a "babysitting app" detects a crash and restarts the crashed app, only to crash again and again. My disk filled with 290GB and GB tar files are inside the AppData/.../sha256/ directory which filled my 1TB disk to 100%. The names of the directories and files are designed so that you do not look at files with meaningless names, do not notice the act of data capture that is mountainous and clumsy. This is part of Docker Desktop 'experimental features'  under settings, so turn that off, but better not use Docker Desktop at all instead WSL with Docker. 
See other bug reports: Related MAJOR BUG:  [Related MAJOR WSL Docker Desktop BUG](https://github.com/microsoft/vscode/issues/181410)
